tools/test-npm.sh

```diff
@@ -29,8 +29,8 @@ export npm_config_prefix="npm-prefix"
 export npm_config_tmp="npm-tmp"
 
 # install npm devDependencies and run npm's tests
-../$NODE_EXE cli.js install --ignore-scripts
-../$NODE_EXE cli.js run-script test-all
+../$NODE cli.js install --ignore-scripts
+../$NODE cli.js run-script test-all
```

Comment:
It's not `./node` because at this point we are inside the `test-npm` directory. :)